argobots.spec

```
@@ -96,6 +101,11 @@ rm -f %{buildroot}%{_libdir}/*.la
 %{_includedir}/*
 
 %changelog
+* Wed Jan 20 2021 B.Faccini <bruno.faccini@intel.com> - 1.1a1-1.g7fd1987
```

Sorry, I might have missed the last time:
```suggestion
* Wed Jan 20 2021 B.Faccini <bruno.faccini@intel.com> - 1.1~a1-1.g7fd1987
```

But you should probably update the date here in any case as it's getting quite stale.  `debian/changelog` also if it has the same date.

---

Ok, and debian/changelog has already been changed for this.